README.md

```diff
@@ -123,17 +123,15 @@ These policies empower SOAP clients to enhance the security of their web service
     - `EncryptionAlgorithm` encryptionAlgorithm : The algorithm to encrypt the SOAP envelope
     - `string` x509Token : The path or token of the X509 certificate
 
-- `AsymmetricBindingConfig`: Represents the record for Username Token with Asymmetric Binding policy.
+- `AsymmetricBindingConfig`: Represents the record for Asymmetric Binding policy.
   - Fields:
-    - `crypto:PrivateKey` signatureKey : The private key to sign the SOAP envelope
-    - `crypto:PublicKey` encryptionKey : The public key to encrypt the SOAP body
-    - `SignatureAlgorithm` signatureAlgorithm : The algorithm to sign the SOAP envelope
-    - `EncryptionAlgorithm` encryptionAlgorithm : The algorithm to encrypt the SOAP body
-    - `string` x509Token : field description
+    - `SignatureConfig` signatureConfig : Configuration for applying digital signatures
+    - `EncryptionConfig` encryptionConfig : Configuration for applying encryption
+    - `string` x509Token : The path or token of the X509 certificate
 
 #### Outbound Security Configurations
 
-- `OutboundSecurityConfig`: Represents the record for outbound security configurations to verify and decrypt SOAP envelopes.
+- `InboundSecurityConfig`: Represents the record for outbound security configurations to verify and decrypt SOAP envelopes.
```

Comment:
This is under the topic `Outbound Security Configurations`. We don't want to change this right? Should be `OutboundSecurityConfig`.